<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphSelectDialog</class>
 <widget class="QDialog" name="graphSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>274</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>448</width>
    <height>274</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Spectrum Selection</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>481</width>
     <height>36</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="selectionBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>511</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>110</height>
    </size>
   </property>
   <property name="title">
    <string>Select Spectral Type</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>471</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="noiseButton">
       <property name="text">
        <string>&amp;Noise Data</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="dataButton">
       <property name="text">
        <string>Transformed &amp;Data</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="spectDataButton">
       <property name="text">
        <string>&amp;Spectral Data</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="filterButton">
       <property name="text">
        <string>&amp;Filter Data</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="weightButton">
       <property name="text">
        <string>Transformed &amp;Weights</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="beamButton">
       <property name="text">
        <string>&amp;Beam Data</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="combinedDataButton">
       <property name="text">
        <string>Weighted &amp;Transfomation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="ensembleButton">
       <property name="text">
        <string>Ensemble Power Spectrum</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Please select the spectrum to be displayed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
